### ISSUE TITLE
Resource proxying demo broken on Edge and IE11

### ISSUE THREAD

**Issue by mmso** [cure53/DOMPurify#279]
> This issue proposes a bug which makes the browser fetch the leaking resources, and not the proxied ones.

### Background & Context

When testing the "Hook to proxy all HTTP leaks including CSS" https://github.com/cure53/DOMPurify/blob/master/demos/hooks-proxy-demo.html the browser is making requests to the leaking http resource.

![ie11](https://user-images.githubusercontent.com/352607/40354023-33e8dfa4-5db3-11e8-9d5e-84daa68f504d.gif)

![edge](https://user-images.githubusercontent.com/352607/40354422-1d2fbfa2-5db4-11e8-8495-dba928dcc6c5.gif)

### Bug

On Chrome, Firefox, Safari it seems to work as expected. But not in IE11 or Edge.

#### Expected output

To only request the proxied resource.

**Comment by cure53** [comment]
Heya,

we believe this to be a browser bug in MSIE11 and Edge. The bug was reported by us in late 2017 but apparently no fix is out yet. 

Not sure if we can do much here.

Cheers,
.mario

**Comment by cure53** [comment]
Yep, just confirmed. This still leaks on latest MSIE11 and Edge.

```javascript
parser=new DOMParser();
parser.parseFromString('<audio src="//evil.com/?parser">', "text/html");

impl=document.implementation.createHTMLDocument('');
impl.body.innerHTML='<bgsound src="//evil.com/?impl"></bgsound>'
```
